lib/modules/manager/terragrunt/index.ts

```diff
@@ -16,7 +16,7 @@ export const categories: Category[] = ['iac', 'terraform'];
 
 export const defaultConfig = {
   commitMessageTopic: 'Terragrunt dependency {{depName}}',
-  fileMatch: ['(^|/)terragrunt\\.hcl$'],
+  filePatterns: ['/(^|/)terragrunt\\.hcl$/'],
```

Comment:
```suggestion
  filePatterns: ['**/terragrunt.hcl'],
```

@rarkins @secustor  Do you think this is easier to understand?

https://playcode.io/2284326